### ISSUE TITLE
GUI start issue on Windows

### ISSUE THREAD

**Issue by teafusion** [mandelbulb-4d/mandelbulb-4d#1]
"The procedure entry point could not be located in the dynamic link library  C:\Windows\SYSTEM32\ig9icd64.dll"

Process seems to happen alright according to terminal, but the GUI fails to start with above message.

**Comment by valera-rozuvan** [comment]
Can you please provide a screenshot of the Command Line messages? Also, what version of OpenGL do you have?

**Comment by teafusion** [comment]
![image](https://user-images.githubusercontent.com/28002449/30282035-c67a4276-971c-11e7-9b78-e135c13036df.png)

**Comment by teafusion** [comment]
![image](https://user-images.githubusercontent.com/28002449/30282058-d656b094-971c-11e7-972f-5496638523fb.png)

**Comment by teafusion** [comment]
![image](https://user-images.githubusercontent.com/28002449/30282102-f817d00a-971c-11e7-8b67-d577f1d779c4.png)

**Comment by valera-rozuvan** [comment]
Are there any error messages below the line "Done. Elapsed time: 4.247000 seconds."? Or is it the last line in the console window?

And please provide a screenshot (or information) about your OpenGL version (see here to discover what's your OpenGL version http://support.esri.com/en/technical-article/000011375 ).

**Comment by teafusion** [comment]
The screenshot shows all the lines. I'll get the OpenCL version next.

**Comment by teafusion** [comment]
OpenGL..
![image](https://user-images.githubusercontent.com/28002449/30282567-53ea1748-971e-11e7-8f31-60caa6c53592.png)

**Comment by teafusion** [comment]
I'm updating graphics drivers currently to see if there's any difference.

**Comment by valera-rozuvan** [comment]
OK. I see that you have an Intel Graphics Card - specifically the Intel HD Graphics 530. There is a known issue (described on the Internet) which affects your card - faulty graphic drivers.

Please see: https://bugs.mojang.com/browse/MC-94900 .

Here is a possible solution for you, described in another project: http://www.minecraftforum.net/forums/support/unmodified-minecraft-client/2627432-1-9-1-10-faqs-read-before-posting .

> This is because of a memory leak caused by Intel's drivers (i.e. faulty drivers). This mainly affects 7th to 9th Generation Intel HD Graphics (ig7xxxxx.dll, ig8xxxxx.dll, ig9xxxxx.dll), which ships with 3rd Generation to 6th Generation Intel Core Processors. In layman's terms, if your Intel CPU is Ivy Bridge (Model number ix-3xxx), Haswell (4xxx), Broadwell (5xxx) or Skylake (6xxx), you may be affected.

> To work around this issue, make sure you have the latest drivers installed and set VBO and VSync to On in Video Settings (Options -> Video Settings -> Use VSync/Use VBO -> On).

**Comment by teafusion** [comment]
Updated drivers did the trick, all works fine now!
